src/controller/AutoCommissioner.cpp

```diff
@@ -661,7 +677,8 @@ DeviceProxy * AutoCommissioner::GetDeviceProxyForStep(CommissioningStage nextSta
 CHIP_ERROR AutoCommissioner::PerformStep(CommissioningStage nextStage)
 {
     DeviceProxy * proxy = GetDeviceProxyForStep(nextStage);
-    if (proxy == nullptr)
+    if ((!mParams.GetNonConcurrentCommissioning().ValueOr(false) || nextStage != CommissioningStage::kFindOperational) &&
```

Comment:
Because when using non-concurrent commissioning, the device_proxy will be null in step FindOperational, the PASE Session device proxy should be released after step `Joining Operational Network`.

Reply:
> the PASE Session device proxy should be released after step Joining Operational Network

Why?